en/components/treegrid/tree_grid.md

```diff
@@ -236,6 +236,38 @@ The indentation of the **tree cells** persists across other tree grid features l
 
 <div class="divider--half"></div>
 
+### Keyboard navigation
+
+Keyboard navigation is available by default in the Tree Grid and aims at covering as many as possible features and scenarios for the end user. When you focus a specific cell and press one of the following key combinations, the described behaviour is performed:
+
+ - `Arrow Up` - navigates one cell up (no wrapping)
+ - `Arrow Down` - navigates one cell down (no wrapping)
+ - `Arrow Left` - navigates one cell left (no wrapping between lines)
+ - `Arrow Right` - navigates one cell right (no wrapping between lines)
+ - `Ctrl + Arrow Up` - navigates to the first cell in the current column;
+ - `Ctrl + Arrow Down` - navigates to the last cell in the current column;
+ - `Ctrl + Arrow Left` -  moves to leftmost cell in row
+ - `Home` - moves to leftmost cell in row
+ - `Ctrl + Home` - moves to leftmost cell in row
+ - `Ctrl + Arrow Right` - moves to rightmost cell in row
+ - `End` - moves to rightmost cell in row
+ - `Ctrl + End` - moves to rightmost cell in row
```

Comment:
Probably using one of the bellow descriptions is more accurate:
`Ctrl + End` - moves to bottom right cell in the grid
or
`Ctrl + End` - moves to rightmost cell of the last row

Reply:
We should change that for Grid and Hierarchical Grid too.